### DISCUSSION TITLE
Feature Request – ATP and ATCR

### DISCUSSION THREAD

**Discussion by Sada3105** [community/community#37717] [New to GitHub]
Before every project deployment we undergo regressive testing by our Development team, Business System Analysts, Quality Assurance team and also Business users who perform UAT. 
The ‘Test Plans’ section in Azure DevOps is widely used for test management to store all the test cases created by several teams and perform their testing. 
However, there are several testing/business processes which are followed and tracked outside of the Azure DevOps. It would be of extreme help if the proposed templates could be incorporated within Azure DevOps.

1.	AZDO Test Plan (ATP)
Here I am suggesting to add few more tabs in the ‘Create New Test Plan’ section to provide more details on the Test Plan being created. 
Please follow the attached ATP Proposal template attached to this email for details. This will provide more insight on the project to be tested, objectives for testing, status of testing etc by entering the details on every tabs/pages.

2.	AZDO Test Completion Report (ATCR)
This will be an additional menu under the ‘Test Plans’ section similar to progress report but the purpose of the test completion report is to mark the summary of testing and procure approvals from the desired approvers. 
As a testing standard procedure, our QA team creates an MS Word document for Test Completion Report (TCR) after testing completion by all parties to summarize the testing statuses and provide sign off for testing before deployment to Production. 
The idea is to incorporate this process in Azure DevOps itself where the details can be entered in certain tabs and at the end we could initiate and capture approvals. 

Expected Benefits from adding the proposed tabs/pages:
•	The testing teams do not have to focus on extra documentations involved in every project release as everything can be stored with Azure DevOps
•	We can fetch online approvals for Test Completion Report and need to follow up on emails sent to the desired Approvers.
•	The status of testing by every department is visible and easy to track
•	It helps to store the dates for every event happened and we don’t have to check our emails to track any activity.
•	All testing artifacts would be found under the sections of Azure Test Plans.

[ATCR Proposal.xlsx](https://github.com/community/community/files/9893699/ATCR.Proposal.xlsx)
[ATP Proposal.xlsx](https://github.com/community/community/files/9893700/ATP.Proposal.xlsx)

**Comment by queenofcorgis** [comment]
Hi @sada3105,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking to share feedback for this specific topic, you might want to try posting somewhere that focuses on this project, such as the [Azure Community(https://techcommunity.microsoft.com/t5/azure-devops/bd-p/AzureDevOpsForum). The GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
